docs/wip/how-to-rollback.md

```diff
@@ -1,12 +1,115 @@
-## Step by step on how to rollback?
+## ⚠️ Warning: this will only work when we start using VTEX infrastructure to build the images. Is also possible to build with Netlify[] — for rollback in Netlify, you can use it's tools.
 
 
-- How much time it takes?
-- security measures
+# The rollback
+The rollback interaction aims to help you to react faster when having trouble, changing the deploy that is on production thought a safe and tested path. This also allows you to reuse older versions of a storefront, for a recurrent store's campaing for example. 
 
-- How the rollback works with dynamic and static data?
-- Caching? How much time it takes to invalidate caching
+# 📚 Step by step on how to rollback
+This document is the first version of a guide for the rollback process using VTEX Store Framework Jamstack. It also explains the rollback expected behavior.
+We'll start showing how to make a safe deploy — one that is ready to be used for a rollback.
 
+## 👮‍♀️ How to make a safe deploy (Security Measures)
+- Create a new branch from the current master (this is a backup branch):
+```
+git checkout master
+git pull
+git checkout -b backup-branch
+```
+- Merge your PR, this will trigger a new version to go up.
```

Comment:
```suggestion

>ℹ️ *In this example, we named the backup branch as `backup-branch`. However, feel free to name it however you want.*

2. Merge your PR. This will automatically trigger a new version to go up.
```